server/helpers/dummyData/ingredients.js

```diff
@@ -0,0 +1,62 @@
+export default () => {
+  const ingredient = {
+    id: 1,
+    name: 'Onions'
+  };
+  const ingredients = {
+    1: {
+      id: 1,
+      name: 'Green vegetable'
+    },
+    2: {
+      id: 2,
+      name: 'Spinach'
+    },
+    3: {
+      id: 3,
+      name: 'Malt XY'
+    },
+    4: {
+      id: 1,
+      name: 'Salt'
+    }
+  };
+
+  const recipeIngredients = {
```

Comment:
This should be an array of objects instead of an object of object

Reply:
fixed https://github.com/hephzaron/youth-interact/pull/1/commits/748a6e90e4db20f9339f6b9911ebf3ee64a589b4